plugins/amqp_compressed_proof_plugin/amqp_compressed_proof_plugin.cpp

```diff
@@ -253,7 +253,12 @@ void amqp_compressed_proof_plugin::plugin_initialize(const variables_map& option
             EOS_ASSERT(filtered_receivers.size(), chain::plugin_config_exception, "Cannot have empty filter list for compressed-proof");
 
             const boost::filesystem::path amqp_unconfimed_file   = dir / (file_prefix + "-unconfirmed-"s + name + ".bin"s);
-            reliable_amqp_publisher& publisher = my->publishers.emplace_back(tokens[2], tokens[3], "", amqp_unconfimed_file, "eosio.node.compressed_proof_v0");
+            reliable_amqp_publisher& publisher = my->publishers.emplace_back(tokens[2], tokens[3], "", amqp_unconfimed_file,
+                                                                             [](const std::string& err){
+                                                                                elog("AMQP fatal error: ${e}", ("e", err));
+                                                                                appbase::app().quit();
```

Comment:
This is a change in behavior.  Will it work if someone fixes the issue with rabbit and then just restarts nodeos?  I'm a bit concerned that doing this will _almost_ work, but end up skipping a single block.

Reply:
Yes, the message is queued to disk and sent when connection is established to rabbit on restart.